apps/desktop-wallet/src/api/addressQueries.ts

```diff
@@ -41,14 +47,15 @@ export const addressAlphBalanceQuery = ({
   const getQueryOptions = (latestTxHash: AddressBalanceQueryProps['latestTxHash']) =>
     queryOptions({
       queryKey: [...ADDRESS_BALANCE_QUERY_KEYS, 'ALPH', { addressHash, latestTxHash, networkId }],
-      queryFn: async () => {
+      queryFn: async (): Promise<AddressAlphBalancesQueryFnData> => {
         const balances = await client.explorer.addresses.getAddressesAddressBalance(addressHash)
 
         return {
           addressHash,
-          balances: {
-            balance: BigInt(balances.balance),
-            lockedBalance: BigInt(balances.lockedBalance)
+          alphBalances: {
+            totalBalance: BigInt(balances.balance),
+            lockedBalance: BigInt(balances.lockedBalance),
+            availableBalance: BigInt(balances.balance) - BigInt(balances.lockedBalance)
```

Comment:
These values will be calculated only when the query is executed and then they will be cached.

I took the liberty and opportunity to finally rename this confusing `balances` to `totalBalances`. I believe having these 3 values will simply the UI logic a lot, and it will cause less errors in the future (I remember an issue in the past where I thought `balance` was the same as `availableBalance` and a sweep endpoint was broken).